bundles/org.openhab.binding.kostalinverter/src/main/java/org/openhab/binding/kostalinverter/internal/KostalInverterFactory.java

```diff
@@ -81,7 +85,9 @@ public KostalInverterFactory(@Reference final HttpClientFactory httpClientFactor
 
     @Override
     public boolean supportsThingType(ThingTypeUID thingTypeUID) {
-        return thingTypeUID.equals(FIRST_GENERATION_INVERTER)
+    @Override
+    public boolean supportsThingType(ThingTypeUID thingTypeUID) {
```

Comment:
Seems like there had been something wrong during merge. This produces a compiler error.